### ISSUE TITLE
Multiple IPs when using 104.238.130.180

### ISSUE THREAD

**Issue by hadfiiw** [Pugmatt/BedrockConnect#214]
When I use 104.238.130.180 as the forwarding server for the list of DNS zones from the readme file, I get the following output from my home computer...

```
nslookup mco.cubecraft.net
Server:		192.168.5.21
Address:	192.168.5.21#53

Non-authoritative answer:
Name:	mco.cubecraft.net
Address: 104.238.130.180
Name:	mco.cubecraft.net
Address: 173.199.115.154

```

I wasn't expecting the 173.199.115.154 address.  Is 173.199.115.154 another BedrockConnect server?

Thanks

**Comment by Pugmatt** [comment]
Correct, 173.199.115.154 is another BedrockConnect server. The main DNS, 104.238.130.180,  uses round-robin to load balance players into 2 different BedrockConnect instances.

**Comment by hadfiiw** [comment]
@Pugmatt Thanks, makes sense.

So this confirmed I am getting back the correct DNS response but the Switch never redirects to the server dialog - it just joins, in this case, the cube craft server.

I have tried setting the Switch DNS to 104.238.130.180 and 8.8.8.8.  I also run my own DNS server here at the house and have set my DNS server to "Forward only" those domains (in this case, the switch uses my DNS server).  Unfortunately, both configurations don't show the server dialog and just connects us to whatever server we clicked on.

Any suggestions?  

Will take me some time but I can get a sniffer/packet capture set up to see what is going on if we think that will help.

<img width="1108" alt="Screen Shot 2021-08-21 at 10 33 47 AM" src="https://user-images.githubusercontent.com/22086548/130325200-2ff1c9df-580b-4995-93a0-06f4cf8031bf.png">

**Comment by Pugmatt** [comment]
Most common reason of something like this happening is your network might be on IPv6, in which case the Switch might be ignoring the IPv4 DNS set in the settings. If this is the case, you would have two options:
- Set the DNS directly in your router settings, using 2001:19f0:4009:4126:5400:00ff:fe22:0d52 as the primary, and 2001:4860:4860::8888 as the secondary (or any other popular DNS that supports IPv6 for the secondary)
- Or, disable IPv6 on your router, if possible. Here's a tutorial by jbob3mm that shows how to disable this: https://www.youtube.com/watch?v=3FhBEI55__s

**Comment by hadfiiw** [comment]
Hi @Pugmatt 

No IPv6 here but I did figure out that the "Content Filtering" built into my home router (Ubiquity Dream Machine Pro) seems to thinking some domain in this process is malicious.

I am going to run a packet capture monitor of the Nintendo Switch to see what is going on.  Will report back...
